samigo/samigo-app/src/webapp/js/samigo-global.js

```diff
@@ -117,11 +117,13 @@ function initRubricDialog(gradingId, doneText, cancelText, titleText, alertText)
   gradingId = ("" + gradingId).replace(".", "\\.");
 
   const input = document.querySelector(`.adjustedScore${gradingId}`);
-  const previousScore = input.value;
+  let previousScore = input.value;
   let componentFound = findAdhocRubricComponent(gradingIdAux);
 
   $(`#modal${gradingId}`).dialog({
     modal: true,
+    closeOnEscape: false,
```

Comment:
Will this not make the modal close when `ESC` is pressed? If yes, why are you changing that? I think most modals in Sakai have that behavior, the bootstrap modals do that for sure, except when they are missing the `tabindex` attribute.

Reply:
I found on the existing rubrics Samigo modal there's a bug if you close the modal without confirming with the Done button (unsaved Rubric evaluations might be confirmed by Samigo). ESC or X button should at least do the same than the Cancel one, so I find for this case for clarity it's better to just disallow these options. It's a complex screen and after thinking about it, restricting cases might help making it a little bit clearer for instructors (probably not clear enough though haha).

Reply:
Ahh, I see, makes sense to me.